### ISSUE TITLE
Ping time is way off

### ISSUE THREAD

**Issue by vallon** [yonkavalow/PingStatus#5]
I am pinging my local router, and "ping" shows <1ms delay:

```
$ ping -c 1 192.168.1.1
PING 192.168.1.1 (192.168.1.1): 56 data bytes
64 bytes from 192.168.1.1: icmp_seq=0 ttl=64 time=0.497 ms
```

For that ping:

```
22:00:01.784707 IP 192.168.1.146 > ROUTER: ICMP echo request, id 30260, seq 0, length 64
22:00:01.785070 IP ROUTER > 192.168.1.146: ICMP echo reply, id 30260, seq 0, length 64
```

timestamp diff is 0.363ms.

But, PingStatus is showing 62ms ping time.  The tcpdump for that ping is:

```
21:58:31.407803 IP 192.168.1.146 > ROUTER: ICMP echo request, id 60211, seq 0, length 64
21:58:31.408266 IP ROUTER > 192.168.1.146: ICMP echo reply, id 60211, seq 0, length 64
```

Timestamp diff is 0.463ms.

I am also finding the same for Google 8.8.8.8.  Shows 43ms in menu-bar, tcpdump for that ping was:

```
22:04:27.988386 IP 192.168.1.146 > dns.google: ICMP echo request, id 32566, seq 0, length 64
22:04:27.995631 IP dns.google > 192.168.1.146: ICMP echo reply, id 32566, seq 0, length 64
```

Timestamp diff is 7.245ms.  Ping from command-line is showing average 8.692ms for 8 samples, so the timestamp appears accurate; PingStatus seems wrong.

Parameters: host 8.8.8.8, interval 2s, timeout 1s, delay 500ms, loss 30%, samples 1.
Mac 12.6.1 Monterey Apple M1

**Comment by yonkavalow** [comment]
Hi thanks for the feedback. I will look into this.

**Comment by yonkavalow** [comment]
This issue was addressed in version 1.8.64
